pyvrp/cpp/Solution.h

```diff
@@ -188,21 +188,39 @@ class Solution
         Route(ProblemData const &data,
               Visits visits,
               VehicleType const vehicleType);
+
+        // This constructor does *no* validation. Useful when unserialising
+        // objects.
+        Route(Visits visits,
+              Distance distance,
+              Load demand,
+              Load excessLoad,
+              Duration duration,
+              Duration timeWarp,
+              Duration travel,
+              Duration service,
+              Duration wait,
+              Duration release,
+              Duration startTime,
+              Duration slack,
+              Cost prizes,
+              std::pair<double, double> centroid,
+              VehicleType vehicleType);
     };
```

Comment:
Here, and for `Solution` below: I need a way to unserialise that does not require a `ProblemData` instance - because we do not have that available. So I'm restoring the complete object state via this constructor.